packages/vitest/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
```

Comment:
```suggestion
```

Reply:
I implemented your suggestions [here](https://github.com/KnapsackPro/knapsack-pro-js/pull/104#discussion_r1878955894), please lmk if there are any more changes needed to this file

Reply:
Looks good. Thank you.